src/modules/settings/components/settings/global/Emotes.jsx

```diff
@@ -85,7 +85,7 @@ function EmotesModule() {
               {formatMessage(
                 {
                   defaultMessage:
-                    'Emote modifiers allow you to transform emotes in realtime. Wide: <code>w! emoteName</code>, Horizontal Flip: <code>h! emoteName</code>, Vertical Flip: <code>v! emoteName</code>, Zero-Width: <code>z! emoteName</code>, Rotate Left: <code>l! emoteName</code>, Rotate Right: <code>r! emoteName</code>, Cursed: <code>c! emoteName</code>, Party: <code>p! emoteName</code>',
+                    'Emote modifiers allow you to transform emotes in realtime. Wide: <code>w! emoteName</code>, Horizontal Flip: <code>h! emoteName</code>, Vertical Flip: <code>v! emoteName</code>, Zero-Width: <code>z! emoteName</code>, Rotate Left: <code>l! emoteName</code>, Rotate Right: <code>r! emoteName</code>, Cursed: <code>c! emoteName</code>, Party: <code>p! emoteName</code>, Shake: <code>s! emoteName</code>',
```

Comment:
I think we need to find a new way to display this list of modifiers, maybe as a modal? like the one within this file

`Emote modifiers allow you to transform emotes in realtime.`

And the phrase `Emote Modifiers` could be a hyperlink to open said Modal,